src/content/development/building-testing/_index.en.md

```diff
@@ -112,3 +112,30 @@ To jump into a shell in Submariner's standard development environment:
 ```shell
 make shell
 ```
+
+## Known issues
+
+### OSX getopt and sed are incompatible with our scripts
+
+symptoms:
+```
+  flags:ERROR short flag required for (tag) on this platform and "docker buildx build" requires exactly 1 argument
+```
+
+or
+
+```
+sed: 1: "/"DAPPER_RUN_ARGS=/{s/[ ...": extra characters at the end of q command
+docker: invalid reference format: repository name must be lowercase.
+```
+
+This problem is seen in OSX while running make because the OSX shipped `getopt` tool is very limited.
+To fix this issue you need to install gnu-getopt:
+
+`brew install gnu-getopt`
```

Comment:
I’ve reworked https://github.com/submariner-io/shipyard/pull/594 to avoid `getopt` ;-).

Reply:
Oh sorry i think I was misleading, it's something in sed. Getopt is also used by shflags and has the same issue, so feel free to keep getopt implementation if it's better, we will need it anyway